tests/test_html.py

```diff
@@ -16,22 +15,6 @@ def data():
 TOKEN = 'pk.abc123'
 
 
-def test_secret_key_CircleViz(data):
```

Comment:
Can we add these two tests back in, if we reinstate the access token error for mapbox URL style sheets only (above commnet)?